open-source-terms.md

```diff
@@ -0,0 +1,259 @@
+# npm Open-Source Terms
+
+These npm Open Source terms of use (these _Terms_) govern access to
+and use of <https://www.npmjs.com> (the _Website_) as well as the
+"npm Public Registry" at <https://registry.npmjs.com> (the _Public
+Registry_). Both the Website and the Public Registry are operated by
+npm, Inc. (_npm_). The Website and the Public Registry are referred to
+together as _npm Open Source_ throughout these Terms.
+
+These npm Open Source Terms were last updated on November 9, 2015.
```

Comment:
The same thought crossed my mind. Unfortunately, i think as a practical matter we do need to make it clear from text of the terms alone when they were last updated.

I suppose we could do some kind of templating or replacement, at the expense of the straightforward fetch-from-GitHub approach we're using now.